salt/metalk8s/ui/deployed.sls

```diff
@@ -45,7 +45,7 @@ Create metalk8s-ui ConfigMap:
     - context: {{ context }} 
     - data:
         config.json: |
-          {"url": "https://{{ control_plane_ip }}:6443"}
+          {"url": "https://{{ control_plane_ip }}:6443", "url_salt": "http://{{ control_plane_ip }}:4507"}
```

Comment:
Please rework this structure at some point.

Reply:
will be managed in another PR later

Reply:
Issue #993

Reply:
@NicolasT what do you mean by rework? renaming "url" and "url_salt" is enough?